File: trunk/src/QFSvb/QFSvb/FileSystem/Dir.vb
Namespace QFSvb.FileSystem

    Public Class Dir
        Inherits FSElement

        Private dPadre As Dir

        Public Property padre() As Dir
            Get
                Return Me.dPadre
            End Get
            Set(ByVal value As Dir)
                Me.dPadre = value
            End Set
        End Property
        ''' <summary>
        ''' Constructor por defecto
        ''' </summary>
        Sub New()
            MyBase.New()
        End Sub

        Sub New(ByVal idFSElement As Integer)
            MyBase.New(idFSElement)
        End Sub


    End Class

End Namespace

File: src/QFS/QFS/FileSystem/Dir.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QFS.FileSystem
{
    class Dir : FSElement
    {
        private Dir dPadre;

        /// <summary>
        /// Constructor por defecto
        /// </summary>
        public Dir() { 
        }

        /// <summary>
        /// Constructor con id
        /// </summary>
        public Dir(int id)
        {
            base(id);
        }

        public Dir padre {
            get { return this.dPadre; }
            set { this.dPadre = value; }
        }
 


    }
}
